figures/README.md

```diff
@@ -72,9 +72,12 @@ This section includes details for file organization and guidelines that are not
 
 ### Individual panels and where to find them
 
-Individual panels, including any standalone legends, should be saved as **PDFs** in `figures/pdfs/<figure>/panels`.
+Individual panels, including any standalone legends, intended for the main text should be saved as **PDFs** in `figures/pdfs/<figure>/panels`.
 Each figure has its own subdirectory within `figures/pdfs`; individual panels for Figure 1 would be saved in `figures/pdfs/fig1/panels`, for Figure 2 in `figures/pdfs/fig2/panels` and so on and so forth.  
 
+Similarly, panels and standalone legends intended for the supplementary materials should be saved as **PDFs** in `figures/pdfs/supp/<figure>/panels`. 
+As for main text figures, individual panels for Figure S1 would be saved in `figures/pdfs/supp/figs1/panels`, and so on.
```

Comment:
Shouldn't all these changes be moving files to `figures/pdfs/supp/figs3`, `figures/pdfs/supp/figs5`, etc. then?

Reply:
Yes, that was done here already. Did I miss something?

Reply:
Sorry, I could have been more specific. There is no `s` in the directory name, so we have `pdfs/supp/fig5/panels`, etc.


<img width="322" alt="Screen Shot 2022-03-27 at 6 09 53 PM" src="https://user-images.githubusercontent.com/19534205/160303094-ae71215d-80ce-46ef-a2ae-4129ee17b890.png">

Look I found a use for the new file tree!

Reply:
Aha, I missed your `s`. Yes, I should change this.